addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -53,7 +53,7 @@ def _compute_mail_attachments_widget(self):
     @api.depends('enable_ubl_cii_xml')
     def _compute_checkbox_ubl_cii_xml(self):
         for wizard in self:
-            wizard.checkbox_ubl_cii_xml = wizard.enable_ubl_cii_xml and wizard.company_id.invoice_is_ubl_cii
+            wizard.checkbox_ubl_cii_xml = wizard.enable_ubl_cii_xml and (wizard.checkbox_ubl_cii_xml or wizard.company_id.invoice_is_ubl_cii)
```

Comment:
we check the UBL checkbox from Peppol module if the peppol checkbox is checked. So, once we uncheck the "Send via Peppol" checkbox, the BIS3 checkbox is deselected here. It would be better to account for the fact that the checkbox might be already selected.

Reply:
I don't like this diff :/

Reply:
why not? I don't see another solution :/

Reply:
if the checkbox is already selected and the `enable_ubl_cii_xml` is True, then we shouldn't deselect it :thinking: I think this is fine